public/API/API_GetUserRankAndScore.php

```diff
@@ -18,7 +18,10 @@
 
 $retVal = [];
 
-$retVal['Score'] = getPlayerPoints($user);
+$retVal['Score'] = 0;
+if (getPlayerPoints($user, $userPoints)) {
+    $retVal['Score'] = $userPoints['RAPoints'] + $userPoints['RASoftcorePoints'];
+}
```

Comment:
I think `Score` should continue to reflect what's displayed in the login section (i.e. hardcore points). After this PR, there's nowhere else where we display the combined value. You could add the breakdown as additional fields if desired.

Reply:
Or keep `Score` as hardcore points and add a `SoftcoreScore` similar to [my suggestion](https://github.com/RetroAchievements/RAWeb/pull/1033/files#r900964715) for `dorequest`.